src/zhinst/toolkit/session.py

```diff
@@ -987,3 +993,24 @@ def server_host(self) -> str:
     def server_port(self) -> int:
         """Server port."""
         return self._daq_server.port
+
+    # Attempt to pass the allow_version_mismatch flag. Fallback in case
+    #  zhinst.core does not support it yet.
+    def _create_daq(
+        self,
+        server_host: str,
+        server_port: int,
+        allow_version_mismatch: bool,
+    ):
```

Comment:
```suggestion
    def _create_daq(
        self,
        server_host: str,
        server_port: int,
        allow_version_mismatch: bool,
    ):
    """Create a new session to the data server.
    
      Attempt to pass the allow_version_mismatch flag. Fallback in case
      zhinst.core does not support it yet."""
```